src/GraphQL/ReadOneAreaResolver.php

```diff
@@ -12,6 +12,10 @@ public function resolve($object, array $args, $context, ResolveInfo $info)
     {
         $area = ElementalArea::get()->byID($args['ID']);
 
+        if (!$area) {
+            throw new \Exception('Could not find elemental area matching ID ' . $args['ID']);
```

Comment:
`InvalidArgumentException` maybe?

Reply:
Using base exception is fairly rampant in GraphQL code. I agree with using specific exceptions though. I've updated it